src/server.cpp

```diff
@@ -2874,8 +2874,14 @@ std::wstring Server::handleChat(const std::string &name, const std::wstring &wna
 				L"It was refused. Send a shorter message";
 	}
 
+	auto message = wide_to_utf8(wmessage);
+	auto first = message.find_first_of("\n\r");
```

Comment:
\r is not used on unix, just check \n

Reply:
it is on Windows, though.

Will logging to file strip away /r?

Reply:
`\r` could produce unexpected results when using `cat` or `tail`:

```
$ echo $'line1\rline2\nline3'
line2
line3
```

Reply:
Maybe check for `\n` _or_ `\r`?

Reply:
That's what this does